client/src/App.js

```diff
@@ -1,24 +1,18 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import GetPlayers from './GetPlayers';
 
 function App() {
+
   return (
     <div className="App">
       <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+        <h1>This Sporting Game Exists</h1>
       </header>
+      <body>
+        <GetPlayers/>
+      </body>
```

Comment:
Not sure but I'm pretty sure there is something against using `body` tags as `div`s try the `section` tag instead